src/index.html

```diff
@@ -7,6 +7,36 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--29 calendar--start-day--sun">
```

Comment:
I guess month length is not working, isn't it?

If you change the "month-length--", the following days should not be automatically displayed in the calendars